opal/util/malloc.c

```diff
@@ -52,13 +52,14 @@ int opal_malloc_debug_level = OPAL_MALLOC_DEBUG_LEVEL;
 int opal_malloc_output = -1;
 
 
+
+#if OPAL_ENABLE_DEBUG
+
 /*
  * Private variables
  */
 static opal_output_stream_t malloc_stream;
```

Comment:
I think it is safe to completely remove malloc_stream, it is never used anywhere.

Reply:
I thought about that as well, but figured this was safest for the purposes of this PR.

Reply:
Removed in a separate commit to make it easier to revert if anyone wants it to hang around.